active/0000-addon-options.md

```diff
@@ -0,0 +1,38 @@
+- Start Date: 2015-03-24
+- RFC PR: 
+- ember-cli Issue: 
+
+# Summary
+
+Enabling addons to specify custom prompts that would inquire the user, and perform arbitrary actions based on those responses. Actions could range from running blueprints, installing additional addons, setting configuration.
+
+Similar to what Yeoman has:
+
+![from yeoman](http://yeoman.io/assets/img/codelab/image_8.5a17.png)
+
+# Motivation
+
+This would pave the way for users creating custom 'app' blueprints that can have their personal/comapany's workflows, with default settings, and optionals. This could include installing the addons you always install, e.g. ember-cli-less, bootstrap, font-awesome, etc.
```

Comment:
Agreed. I was actually thinking of doing something similar with prompts, where a custom prompt exists that installs packages with the same kind of format, i.e. a list of packages.

So a single prompt could install a single packages or several.

> Would you like to setup the default UI frameworks?

Something along the lines of (not covering the bower/npm use case -- not sure I want too lol).
```js
{
 type: 'packages',
 message: 'Would you like to setup the default UI frameworks?',
 packages: [
   'fontawesome',
   'bootstrap',
   'ember-cli-less'
 ]
}
```